contracts/deploy/deployActions.js

```diff
@@ -768,7 +835,157 @@ const deployNativeStakingSSVStrategy = async () => {
 };
 
 /**
- * Deploy the OracleRouter and initialise it with Chainlink sources.
+ * Deploy CompoundingStakingSSVStrategy
+ * Deploys a proxy, the actual strategy, initializes the proxy and initializes
+ * the strategy.
+ */
+const deployCompoundingStakingSSVStrategy = async () => {
+  const assetAddresses = await getAssetAddresses(deployments);
+  const { governorAddr, deployerAddr } = await getNamedAccounts();
+  const sDeployer = await ethers.provider.getSigner(deployerAddr);
+  const networkName = await getNetworkName();
+
+  const cOETHVaultProxy = await ethers.getContract("OETHVaultProxy");
+
+  log("Deploy Beacon Proofs");
+  await deployWithConfirmation("BeaconProofs", []);
+  const cBeaconProofs = await ethers.getContract("BeaconProofs");
+
+  let governorAddress;
+  // Deploy the proxy on Hoodi fork not as defender relayer since we will not
+  // test SSV token claiming on that testnet
+  if ((isTest && !isFork) || networkName == "hoodi") {
+    // For unit tests and Hoodi, use the Governor contract
+    governorAddress = governorAddr;
+
+    log("Deploy CompoundingStakingSSVStrategyProxy");
+    await deployWithConfirmation("CompoundingStakingSSVStrategyProxy");
+  } else {
+    // For fork tests and mainnet deployments, use the Timelock contract
+    governorAddress = addresses.mainnet.Timelock;
+    log(`Mainnet governor is the Timelock contract ${governorAddress}`);
+  }
+
+  let cCompoundingStakingSSVStrategyProxy;
+  if (isTest) {
+    log(`Fix CompoundingStakingSSVStrategyProxy address for unit tests`);
+    // For unit tests, fix the address of compoundingStakingSSVStrategy so the withdrawal credentials
+    // are fixed for the validator public key proofs
+    await replaceContractAt(
+      addresses.mainnet.CompoundingStakingStrategyProxy,
+      await ethers.getContract("CompoundingStakingSSVStrategyProxy")
+    );
+    // Set the governor in storage of the proxy to the deployer
+    await setStorageAt(
+      addresses.mainnet.CompoundingStakingStrategyProxy,
+      "0x7bea13895fa79d2831e0a9e28edede30099005a50d652d8957cf8a607ee6ca4a", // governor storage slot
+      deployerAddr
+    );
+    cCompoundingStakingSSVStrategyProxy = await ethers.getContractAt(
+      "CompoundingStakingSSVStrategyProxy",
+      addresses.mainnet.CompoundingStakingStrategyProxy
+    );
+  } else {
+    // For fork tests, mainnet and Hoodi deployments.
+    // Should have already been deployed by the Defender Relayer as SSV rewards are sent to the deployer.
+    // Use the deployStakingProxy Hardhat task to deploy
```

Comment:
We probably need the defender relayer to deploy an empty proxy for this strategy?